compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -258,23 +268,16 @@ fun CanvasBasedWindow(
         // because the default behaviour expects that the Canvas takes the entire window space,
         // so the app has the same lifecycle as the browser tab.
         window.addEventListener("resize", { _ ->
-            val w = document.documentElement?.clientWidth ?: 0
-            val h = document.documentElement?.clientHeight ?: 0
-            channel.trySend(IntSize(w, h))
+            channel.trySend(getParentContainerBox())
         })
 
+        channel.trySend(getParentContainerBox())
+
```

Comment:
A separate thing that is also important to mention (but not so important to fix) - if we still want async resize, better to subscribe on it inside a separate coroutine instead of `LaunchedEffect`. When we subscribe inside `LaunchedEffect`:
-  a child (ComposeScene) manipulates its parent (Canvas), it can complicate the code in the future (it is simple right now).
- the Coroutine dispatcher for effects is partially frame-bound. It is complicated to explain in 2 words, but the main rule - if a coroutine doesn't affect the Composable content, don't place it inside LauchedEffect.